dev/main.tf

```diff
@@ -2,21 +2,9 @@ variable "bitcoin_network" {}
 variable "name_prefix" {}
 
 locals {
-  bitcoin_network          = var.bitcoin_network
-  name_prefix              = var.name_prefix
-  letsencrypt_issuer_email = "dev@galoy.io"
-}
+  bitcoin_network = var.bitcoin_network
+  name_prefix     = var.name_prefix
 
-module "infra_services" {
-  source = "git::https://github.com/GaloyMoney/galoy-infra.git//modules/services?ref=86b0906"
-
-  name_prefix                 = local.name_prefix
-  letsencrypt_issuer_email    = local.letsencrypt_issuer_email
-  local_deploy                = true
-  cluster_endpoint            = "dummy"
-  cluster_ca_cert             = "dummy"
-  honeycomb_api_key           = "dummy"
```

Comment:
@openoms i am trying to configure my local dev charts environment to use my honeycomb api keys. Is there a new way to do this now?

Reply:
The setting is here now:
https://github.com/GaloyMoney/charts/blob/d526361517dd559c7669a0b210b5842666e88b6c/charts/galoy-deps/values.yaml#L23
can you try filling that?